shell/browser/ui/views/menu_bar.h

```diff
@@ -92,6 +92,7 @@ class MenuBar : public views::AccessiblePaneView,
   raw_ptr<NativeWindow> window_;
   raw_ptr<RootView> root_view_;
   raw_ptr<ElectronMenuModel> menu_model_ = nullptr;
+  raw_ptr<MenuDelegate> menu_delegate_ = nullptr;
```

Comment:
Shouldn't this be a `unique_ptr` ? `raw_ptr` does not manage ownership. `menu_delegate_` is leaked with this change.

Reply:
Yeah, you are right, I will change it in next commit